WordPress/Classes/ViewRelated/Reader/ReaderSiteStreamHeader.xib

```diff
@@ -1,191 +1,121 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.XIB" version="3.0" toolsVersion="10117" systemVersion="15F34" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES">
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think the `didTapFollowButton` IBAction in here needs reconnecting – the method is called just fine, but IB shows it as a warning, and the .swift file shows the action as not connected.